cmd/soroban-cli/src/strval.rs

```diff
@@ -318,13 +321,13 @@ impl Spec {
                 let val = self.from_json(v, &f.type_)?;
                 let key = StringM::from_str(name).unwrap();
                 Ok(ScMapEntry {
-                    key: ScVal::Symbol(key),
+                    key: ScVal::Symbol(key.try_into().map_err(|_| Error::Unknown)?),
```

Comment:
Can we do a better error here than Unknown? Do we have symbol parsing errors we can re-use?

Edit; Same below were we `.try_into()` symbols.